istioctl/pkg/writer/envoy/configdump/configdump.go

```diff
@@ -121,26 +121,32 @@ func (c *ConfigWriter) PrintSecretSummary() error {
 }
 
 func (c *ConfigWriter) PrintFullSummary(cf ClusterFilter, lf ListenerFilter, rf RouteFilter, epf EndpointFilter) error {
+	_, _ = c.Stdout.Write([]byte("------ BOOTSTRAP INFO ------\n\n"))
 	if err := c.PrintBootstrapSummary(); err != nil {
 		return err
 	}
 	_, _ = c.Stdout.Write([]byte("\n"))
+	_, _ = c.Stdout.Write([]byte("------ CLUSTER INFO ------\n\n"))
 	if err := c.PrintClusterSummary(cf); err != nil {
 		return err
 	}
 	_, _ = c.Stdout.Write([]byte("\n"))
+	_, _ = c.Stdout.Write([]byte("------ LISTENER INFO ------\n\n"))
```

Comment:
out of curiousity, is there a specific reason you did two newlines instead of one?